lua/barbar/events.lua

```diff
@@ -30,6 +30,9 @@ local render = require'barbar.render'
 local state = require'barbar.state'
 local utils = require'barbar.utils'
 
+--- The `<mods>` used for the close click handler
+local CLOSE_CLICK_MODS = vim.api.nvim_cmd and {confirm = true} or 'confirm'
```

Comment:
We can remove this if we up the minimum Neovim version in v2.

Reply:
lint: space inside braces

Reply:
Right! I really have to get the formatter working.